maths/arc_length.py

```diff
@@ -13,3 +13,7 @@ def arc_length(angle: int, radius: int) -> float:
 
 if __name__ == "__main__":
     print(arc_length(90, 10))
+    """
+    >>> arc_length(90, 10)
+    15.7079632679
```

Comment:
This should be placed with the rest of the doctests, not separately in the `__main__` block